store/src/java/com/zimbra/util/ExternalVolumeInfoHandler.java

```diff
@@ -67,6 +83,41 @@ public JSONObject readServerProperties(int volumeId) throws ServiceException, JS
         return retJsonObj;
     }
 
+    /**
+     * Read backup server level external volume properties
+     * @param volumeId
+     * @param filePath
+     *
+     * @return JSONObject as a backup server level volume properties
+     * @throws JSONException, ServiceException
+     */
+    public JSONObject readBackupServerProperties(int volumeId, String filePath) throws ServiceException, JSONException {
+        JSONObject retJsonObj = new JSONObject();
+        try {
+            // step 1: Fetch current JSON state object and current JSON state array
+            String serverExternalStoreConfigJson = readServerJSONFile(filePath);
```

Comment:
serverExternalStoreConfigJson could be null

Reply:
Initialized to empty in-order to avoid null pointer exception